cmd/clusterctl/clusterdeployer/bootstrap/options.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bootstrap
+
+import "github.com/spf13/pflag"
+
+type Options struct {
+	Type             string
+	Cleanup          bool
+	MiniKubeFlags    []string
+	MinikubeVmDriver string
+	KubeConfig       string
+}
+
+func (o *Options) AddFlags(fs *pflag.FlagSet) {
+	fs.StringVarP(&o.Type, "bootstrap-type", "", "minikube", "The cluster bootstrapper to use")
```

Comment:
it was discussed in the meeting today the we shouldn't have a minikube (or even kind) as a default option. possibly the default option should be "none" (in a way).
could we instead break the CLI execution and say something like "hey, you didn't provide a kubeconfig and didn't provide a boostrapper".

Reply:
Fixed, great feedback, thank you!